pkg/patterns/declarative/reconciler.go

```diff
@@ -656,6 +719,9 @@ func (r *Reconciler) CollectMetrics() bool {
 	return r.options.metrics
 }
 
+// GetObjectFromCluster gets the current state of the object from the cluster.
+//
+// deprecated: use LiveObjectsFunc instead when computing status
```

Comment:
Do you mean the `LiveObjectReader` in this PR or in some code in other feature branch? I can't find a `LiveObjectFunc` else where [in the master k-d-p branch ](https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/search?q=LiveObjectsFunc)

Reply:
Good point - I think I renamed it a few times - I do mean LiveObjectReader.  Thank you!